src/traits/window.rs

```diff
@@ -6,8 +6,21 @@ use glib::translate::{from_glib_none, ToGlibPtr};
 use ffi;
 use glib::to_gboolean;
 use cast::GTK_WINDOW;
+use gdk;
 
 pub trait WindowTrait : ::WidgetTrait {
+    fn move_(&self, x: i32, y: i32) {
```

Comment:
@gsingh93: Don't worry, I just thought about it when I read your code. =D
@gkoz: Where did you find the `isize` size ? I thought it was supposed to be the "replacement" for C `int` types. I'm quite surprised.